server/src/lib/webhooks/handleWebhooks.ts

```diff
@@ -0,0 +1,47 @@
+import { prisma } from '../../prisma';
+import { Request } from 'express';
+import { createAppAuth } from '@octokit/auth-app';
+import { AppAuthentication } from '@octokit/auth-app/dist-types/types';
+import { config } from '../../config';
+import { deployAppQueue } from '../../queues/deployApp';
+
+export const handleWebhooks = async (req: Request) => {
+  if (!req.body) {
+    throw new Error('Failed to fetch the request from github');
+  }
+
+  const auth = createAppAuth({
+    appId: config.githubAppId,
+    privateKey: config.githubAppWebhookSecret,
+    clientId: config.githubAppClientSecret,
+    clientSecret: config.githubAppClientSecret,
+  });
+
+  const installationAuthentication = (await auth({
+    type: 'installation',
+    installationId: req.body.installation.id.toString(),
+  })) as AppAuthentication;
+
+  const appsToRedeploy = await prisma.appMetaGithub.findMany({
+    where: {
+      repoId: req.body.repository.id.toString(),
+      repoOwner: req.body.repository.owner.name,
```

Comment:
do you need the other fields in the findMany? I think just repoId is good enough no?